mollie/api/objects/payment.py

```diff
@@ -146,3 +147,43 @@ def settlement_amount(self):
     @property
     def method(self):
         return self._get_property('method')
+
+    @property
+    def value(self):
+        return self.amount['value']
+
+    @property
+    def currency(self):
+        return self.amount['currency']
+
+    @property
+    def order_id(self):
```

Comment:
Metadata is free-form, je kunt er alles in stoppen. Het heeft geen zin om proxy methodes aan te maken voor onderdelen daarvan.